BUILD.md

```diff
@@ -43,6 +43,27 @@ commands (or set them to `true` to disable C compilation altogether)
 if you don't want `sbt compile` to call your C compiler with `cc` and
 `ar` as the default commands.
 
+## CentOS 7
+
+You can use the `yum` package manager to install most of the tools
+needed to build Daffodil:
+
+    sudo dnf install gcc git java-11-openjdk-devel llvm make pkgconfig
```

Comment:
Should this be yum instead of dnf? Does CentOS 7 have dnf? Should this also be gcc instead of clang like we suggest for fedora, or is clang not even available?